File: resources/algorithms/sorting/bubble_sort.cpp
// NAME: Cecilio Navarro
// ORGN: 
// FILE: bubble_sort.cpp
// DATE: Fri May 24 2024

#include <iostream>

using namespace std;

int main() {
    const int arraySize = 5;
    int array[] = {5, 24, 7, 2, 10};
    for (int i=0; i<arraySize; i++) {
        for (int j=0; j<arraySize; j++) {
            if (array[j] < array[j+1]) {
                int temp = array[j];
                array[j] = array[j+1];
                array[j+1] = temp;
            }
        }
    }
    for (int i=0; i<arraySize; i++) {
        cout << array[i] << "\n";
    }
}

File: resources/algorithms/sorting/bubble/bubble_sort.py
# NAME: Cecilio Navarro
# ORGN: resources/algorithms/sorting/bubble_sort.py
# FILE: bubble_sort.py
# DATE: Fri May 24

def bubbleSort(my_array):
    arraySize = len(my_array)
    for i in range(arraySize):
        for j in range(0, my_array-i-1):
            if (my_array[j] < my_array[j+1]):
                my_array[j], my_array[j+1] = my_array[j+1], my_array[j]
    return my_array

my_array = [5, 4, 3, 2, 1]

sorted_array = bubbleSort(my_array)

print(sorted_array)

